### DISCUSSION TITLE
ScriptReference/AssetDatabase.IsSubAsset

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#566] [Notes]
# ScriptReference/AssetDatabase.IsSubAsset

https://docs.unity3d.com/ScriptReference/AssetDatabase.IsSubAsset.html

**Comment by electronics45** [comment]
This method DOES NOT WORK if the target sub asset has the HideInHierarchy hide flag set (e.g. sub assets of an AnimatorController asset).
use `!AssetDatabase.IsMainAsset()` for a reliable way to test for sub assets.
